README.md

```diff
@@ -16,20 +25,40 @@ For updates, help, questions, feedback and other requests related to this packag
 
 ### Motivation
 
+This module is suitable for storing big amount of records in the regions without affecting heap memory.
+Can be used for logs, events history etc.
+
 ### Interface
 
+```motoko
+stable let log : StableLogLists.StableLogLists = StableLogLists.new();
```

Comment:
Can we shorten this so that user can write `StableLogLists.LogLists`. Internally we don't really need the word stable. Should be enough if the package name has that word.